file-formats/edcc/type/zif_aff_edcc_v1.intf.abap

```diff
@@ -0,0 +1,552 @@
+      "! Mismatch field group description
+      "! $required
+      description       TYPE ty_short_description,
+      "! <p class="shorttext">Level</p>
+      "! Level of comparison field group
+      "! $required
+      level             TYPE ty_level,
+      "! <p class="shorttext">Check Formula</p>
+      "! Formula applied to perform the content check on the field group
+      "! $required
+      check_formula     TYPE ty_check_formula,
+      "! <p class="shorttext">Result Process</p>
+      "! Result process
+      "! $required
+      result_process    TYPE ty_result_process,
+      "! <p class="shorttext">Comparison Fields</p>
+      "! Field Names whose values will be compared
+      "! $required
+      comparison_fields TYPE ty_comparison_fields,
+    END OF ty_comparison_field_group.
+  "! <p class="shorttext">Comparison Field Groups</p>
+  "! Comparison field groups
+  TYPES ty_comparison_field_groups TYPE SORTED TABLE OF ty_comparison_field_group WITH UNIQUE KEY name.
+  TYPES:
+    "! <p class="shorttext">Content Check</p>
+    "! Content check
+    BEGIN OF ty_content_check,
+      "! <p class="shorttext">Check ID</p>
+      "! Check id
+      "! $required
+      check_id                TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Check Number</p>
+      "! Check number
+      "! $required
+      check_number            TYPE ty_check_number,
+      "! <p class="shorttext">Additional Data Source</p>
+      "! Additional data source
+      data_source             TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Additional Description</p>
+      "! Additional check description
+      "! $required
+      additional_description  TYPE zif_aff_types_v1=>ty_description_60,
+      "! <p class="shorttext">Comparison Field Groups</p>
+      "! Comparison field groups
+      "! $required
+      comparison_field_groups TYPE ty_comparison_field_groups,
+    END OF ty_content_check.
+  "! <p class="shorttext">Content Checks</p>
+  "! Assigned content checks
+  TYPES ty_content_checks    TYPE SORTED TABLE OF ty_content_check WITH UNIQUE KEY check_id check_number.
+  "! <p class="shorttext">Sequence</p>
+  "! Check run sequence
+  TYPES ty_run_sequence      TYPE c LENGTH 3.
+  "! <p class="shorttext">Control Parameter</p>
+  "! Check execution control parameter
+  "! $values {@link zif_aff_edcc_v1.data:co_control_parameter}
+  "! $default {@link zif_aff_edcc_v1.data:co_control_parameter.always_run_check}
+  TYPES ty_control_parameter TYPE c LENGTH 2.
+  TYPES:
+    "! <p class="shorttext">Run Attribute</p>
+    "! Check run attribute
+    BEGIN OF ty_run_attribute,
+      "! <p class="shorttext">Check ID</p>
+      "! Check id
+      "! $required
+      check_id          TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Sequence</p>
+      "! Check run sequence
+      "! $required
+      run_sequence      TYPE ty_run_sequence,
```

Comment:
Minor: Mart this as resolved, too. Again, only feedback:

For me this is not the sequence, but the position within the sequence